openwisp_monitoring/device/admin.py

```diff
@@ -109,15 +153,44 @@ class MetricInline(NestedGenericStackedInline):
     verbose_name = _('Alert Settings')
     verbose_name_plural = verbose_name
 
+    def _set_alertsettings_inline(self):
+        self.fields = ['name', 'is_healthy', 'field_name', 'configuration']
+        self.readonly_fields = ['is_healthy']
+
```

Comment:
Similar improvements here to use `get_fields` and `get_readonly_fields` here

Reply:
I'll update that :+1:  thanks :)